src/PluginClient.php

```diff
@@ -79,7 +78,7 @@ public function sendRequest(RequestInterface $request)
         $pluginChain = $this->createPluginChain($this->plugins, function (RequestInterface $request) {
             try {
                 return new FulfilledPromise($this->client->sendRequest($request));
-            } catch (HttplugException $exception) {
+            } catch (\Exception $exception) {
```

Comment:
what situations do we have non-http exceptions being thrown around? is that an expected behaviour that the application should handle, or is it only on configuration errors and the like? on misconfiguration, the plugins should not need to act, the developer will have to fix the bug he or she created. not going through the plugin chain in such cases will make debugging easier actually.